live-examples/html-examples/global-attributes/meta.json

```diff
@@ -32,6 +32,17 @@
             "title": "HTML Demo: dir",
             "type": "tabbed",
             "height": "tabbed-standard"
+        },
+        "tabindex": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode":
+                "live-examples/html-examples/global-attributes/tabindex.html",
+            "cssExampleSrc":
+                "live-examples/html-examples/global-attributes/css/tabindex.css",
+            "fileName": "tabindex.html",
+            "title": "HTML Demo: tabindex",
+            "type": "tabbed",
+            "height": "tabbed-standard"
```

Comment:
Should also be "tabbed-shorter". I feel like I missed a bunch of these in my first reviews today. Oh well.